src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -66,11 +66,13 @@ class NewFurniture extends React.Component {
     for (let page = 0; page < pagesCount; page++) {
       swipeContent.push(
         <div className='row'>
-          {categoryProducts.slice(activePage * 8, (activePage + 1) * 8).map(item => (
-            <div key={item.id} className='col-3'>
-              <ProductBox {...item} />
-            </div>
-          ))}
+          {categoryProducts
+            .slice(activePage * itemsPerPage, (activePage + 1) * itemsPerPage)
```

Comment:
Dlaczego przeniosłeś .slice(...) do nowej linii?

Reply:
@akijania To jest akurat poprawnie sformatowany fragment